README.md

```diff
@@ -1,51 +1,38 @@
-==========================================================================
-GeoToad %VERSION% by Thomas Stromberg and The GeoToad Project (c) 2002 - 2016
-==========================================================================
-
-Table of Contents:
-------------------
-* About GeoToad
-* System Requirements
-* Getting Started
-* Command-Line Syntax
-* Command-Line Examples
-
-
--------------
-About GeoToad
--------------
-GeoToad is an open-source query tool to query the geocaching.com website.
-It supports exporting your results into a dozen or more formats such as
-GPX (also variants suited for GSAK, or your road nav device) and HTML.
-
-
--------------------
-System Requirements
--------------------
- - Windows
-   - Windows 98 or Higher with
-     - a sufficiently recent Ruby version
-     - a SSL certificate chain required for HTTPS (also provided with GeoToad)
- - Mac OS
-   - Mac OS X 10.4 or Higher
- - Other Operating Systems
-   - You may need to install Ruby before running GeoToad.
-     - RedHat/Fedora Linux: built-in
-     - Debian/Ubuntu Linux: apt-get install ruby libopenssl-ruby, then dpkg -i geotoad_*.deb
-     - FreeBSD: pkg_add -r ruby
-
-Optionally, if you have gpsbabel (=> http://www.gpsbabel.org/)
-GeoToad can output to over a dozen additional GPS formats.
-
-
----------------
-Getting Started
----------------
-The easiest way to get started with GeoToad is to use the "text user
-interface" (TUI). Simply double click on the GeoToad program icon, or from
-a command prompt run "geotoad". On UNIX, you may have to run "./geotoad.rb".
+# GeoToad
+by Thomas Stromberg and The GeoToad Project © 2012–2016
+
+## Table of Contents:
+* [About GeoToad](#about-geotoad)
+* [System Requirements](#system-requirement)
+* [Getting Started](#getting-started)
+* [Command-Line Syntax](#command-line-syntax)
+* [Command-Line Examples](#command-line-exampes)
+
+## About GeoToad
+GeoToad is an open-source query tool to query the geocaching.com website. It supports exporting your results into a dozen or more formats such as GPX (also variants suited for GSAK, or your road nav device) and HTML.
+
+## System Requirements
+#### Windows:
+* Windows 98 or Higher with
+  * a sufficiently recent Ruby version
+  * a SSL certificate chain required for HTTPS (also provided with GeoToad)
+
+#### macOS
```

Comment:
Another note: Did you notice that a README.md already exists (in the wiki section) which has to be "textified" to produce the README file in the tarball? (A md2txt converter would be highly appreciated.)
Yes, there is some duplication, due to the later addition of wiki pages - which now are supposed to be the primary source of information. People won't be happy to off-line read md pages, so we'll probably have to keep the dupes.